src/blog/_posts/2023-06-08-survey-results.md

```diff
@@ -0,0 +1,54 @@
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We have released a user survey earlier this year to learn about who our users were, how they use Cadence and how we could help them. After collecting your great feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable to us and it helps us shape our roadmap for the future.
+
+Here are some highlights in text and you can check out the visuals to get more details:
+
+![using.png](../../shared/img/2023-survey-results/using.png)
+
+![job_role.png](../../shared/img/2023-survey-results/job_role.png)
+
+Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace to get any guidance.
+
+![scale.png](../../shared/img/2023-survey-results/scale.png)
+
+Scale our users operating Cadence varies from thousands to billions of workflows per month. It was exciting to see it being used in both small and large scale companies.
+
+![time_zone.png](../../shared/img/2023-survey-results/time_zone.png)
+
+Our user base is the biggest in Europe compared to any other place. Several of those users contacted us to contribute to Cadence as a follow up to the survey. We will start putting up-for-grabs tasks on Github. Several of them wanted to meet with a Zoom call and to discuss their use cases and best practices. As the Cadence team has presence in both the EU and the US, we welcome all our users to contact us anytime. Slack is the fastest way to reach.
+
+![following.png](../../shared/img/2023-survey-results/following.png)
+
+![channels.png](../../shared/img/2023-survey-results/channels.png)
+
+Cadence is followed in [Slack](https://uber-cadence.slack.com/) the most, then [Github](https://github.com/uber/cadence) and [LinkedIn](https://www.linkedin.com/company/cadenceworkflow/). We are the most active in Slack and we plan to be more active in other mediums as well.
+
+![scenarios.png](../../shared/img/2023-survey-results/scenarios.png)
+All of our main use cases were used across the board. While we mentioned the most common cases, several others were mentioned as a comment: enhanced timers, leader election etc.
+
+We found out that Cadence has been used in several science communities. Some of them were using community built clients and were asking if we are going to support more languages. We are planning to take the ownership of Python and Javascript/Typescript clients and support them officially.
+
+![improvement.png](../../shared/img/2023-survey-results/improvement.png)
+
+Documentation is by far what our users wanted improvements on. We are revamping our documentation soon, and expect to see major changes in our website soon.
```

Comment:
Documentation is by far what our users wanted improvements on. We are revamping our documentation and there will be major changes on our website soon.